src/backend/base/langflow/components/prototypes/JSONCleaner.py

```diff
@@ -0,0 +1,87 @@
+import json
+import json_repair #type: ignore
```

Comment:
can you put it in the `clean_json` method? with safe try/except

similar to https://github.com/langflow-ai/langflow/blob/74433bf023a66a0e7b98cb8eec0f14a1f39e5404/src/backend/base/langflow/components/vectorstores/AstraDB.py#L170-L177

Reply:
All done! :)